project_reach/lib/dataset_specification.py

```diff
@@ -0,0 +1,32 @@
+from core_data_modules.cleaners import somali
+
+
+class CodingPlan(object):
+    def __init__(self, source_field, auto_coded_field, manually_coded_field, coda_name, cleaner=None):
+        self.source_field = source_field
+        self.auto_coded_field = auto_coded_field
+        self.manually_coded_field = manually_coded_field
+        self.coda_name = coda_name
+        self.cleaner = cleaner
+
+
+class DatasetSpecification(object):
```

Comment:
This name and content doesn't quite align to me, esp. without any commentary as to what this object is. Perhaps we need to talk in person?

Reply:
I was wondering if it would be better to move the project-specific constants which are spewed around the rest of the project in to one place (here). The current contents made the most sense to move first, due to the duplication across files, but other constants could go here too e.g. configuration of the analysis keys, which are separately named here and in analysis_file.py